### PR TITLE
[General] Updating info on Resources page

### PR DESCRIPTION
**Pull request by mscharlock**
Adds MZ's book, LinkedIn/Pluralsight courses

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3a4f4da](https://github.com/OfficeDev/office-js-docs-pr/pull/1039/commits/3a4f4da65b908f0bbc5b3d8882db330d077c1450)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/resources-links-help.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-resourcesupdate/docs/resources/resources-links-help.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905281833276639-1039%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=P7Q%2FQL%2BNgon2lEXwUHoQ8WKxFPT5mJtSXM8fr7d6ChM%3D&st=2019-05-28T18%3A29%3A57Z&se=2019-06-28T18%3A34%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/resources/resources-links-help.md [hunk 1]
```diff
@@ -16,3 +16,7 @@ These resources provide additional information and support for developing Office
...
 |**Documentation updates**| To make changes to the documentation yourself, choose the **Edit** link on an article and submit a pull request to the <a href="https://github.com/officedev/office-js-docs-pr" target="_blank">OfficeDev/office-js-docs-pr</a> GitHub repository or to the <a href="https://github.com/officedev/outlook-dev-docs" target="_blank">OfficeDev/outlook-dev-docs</a> GitHub repository (for Outlook documentation updates).
+|**Pluralsight Course about Office Add-ins**| <a href="https://www.pluralsight.com/courses/build-office-addins-js-api" target="_blank">John Brown's course "Fundamentals of Building Office Add-ins with Office JavaScript APIs"</a> teaches you how to develop a PowerPoint Add-in from scratch, including information about how to communicate with Trello and publish the add-in to the Microsoft Office Store.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md]: Remember to update the ms.date

Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md]:
```suggestion
|**Pluralsight course about Office Add-ins**| <a href="https://www.pluralsight.com/courses/build-office-addins-js-api" target="_blank">John Brown's course "Fundamentals of Building Office Add-ins with Office JavaScript APIs"</a> teaches you how to develop a PowerPoint Add-in from scratch, including information about how to communicate with Trello and publish the add-in to the Microsoft Office Store.
```

File: docs/resources/resources-links-help.md [hunk 2]
```diff
@@ -16,3 +16,7 @@ These resources provide additional information and support for developing Office
...
+|**LinkedIn Course about Office Add-ins**| <a href="https://www.linkedin.com/learning/microsoft-office-add-ins-for-developers/microsoft-office-add-ins?u=3322">Bill Ayer's course "Microsoft Office Add-Ins for Developers"</a> gives an overview of the platform and explains many of the major concepts of Office Add-ins.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md]:
```suggestion
|**LinkedIn course about Office Add-ins**| <a href="https://www.linkedin.com/learning/microsoft-office-add-ins-for-developers/microsoft-office-add-ins?u=3322">Bill Ayer's course "Microsoft Office Add-Ins for Developers"</a> gives an overview of the platform and explains many of the major concepts of Office Add-ins.
```

File: docs/resources/resources-links-help.md [hunk 3]
```diff
@@ -16,3 +16,7 @@ These resources provide additional information and support for developing Office
...
+|**Building Office Add-ins using Office.js Book**| <a href="https://leanpub.com/buildingofficeaddins">Michael Zlatkovsky's Building Office Add-ins using Office.js book</a> describes the principles and design patterns shared by the 2016 APIs for Excel, Word, and OneNote. Michael is a member of the Office Extensibility team at Microsoft and a Microsoft MVP, but this book is independently authored without input from Microsoft. Neither Microsoft nor Michael collect any profit on this book, as proceeds are donated to disaster-relief and humanitarian charitable causes.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/resources-links-help.md]:
```suggestion
|***Building Office Add-ins using Office.js* book**| <a href="https://leanpub.com/buildingofficeaddins">Michael Zlatkovsky's book *Building Office Add-ins using Office.js*</a> describes the principles and design patterns shared by the 2016 APIs for Excel, Word, and OneNote. Michael is a member of the Office Extensibility team at Microsoft and a Microsoft MVP, but this book is independently authored without input from Microsoft. Neither Microsoft nor Michael collect any profit on this book, as proceeds are donated to disaster-relief and humanitarian charitable causes.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[7f59dc8](https://github.com/OfficeDev/office-js-docs-pr/pull/1039/commits/7f59dc8fd43ce607f93f639496b7e64dc5ee440d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/resources-links-help.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-resourcesupdate/docs/resources/resources-links-help.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905292155548454-1039%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=G99UpSY0SJbmrgtoMV%2FdbZYr69zzEV5PYj4QfrTszbc%3D&st=2019-05-29T21%3A51%3A54Z&se=2019-06-29T21%3A56%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[59daee9](https://github.com/OfficeDev/office-js-docs-pr/pull/1039/commits/59daee93fbf27c0f686bfc41919512b53a9c0912)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/resources-links-help.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-resourcesupdate/docs/resources/resources-links-help.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905292159375435-1039%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2BVTfaX9eJw2BxoQdn%2F3DTlfNxBkcFtQgMntUG5i1RJE%3D&st=2019-05-29T21%3A55%3A47Z&se=2019-06-29T22%3A00%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**